src/test/java/tests/client/CreateOrderTest.java

```diff
@@ -0,0 +1,58 @@
+package tests.client;
+
+import org.testng.Assert;
+import org.testng.annotations.*;
+import pages.HomePage;
+import pages.MenuPage;
+import pages.RestaurantsPage;
+import pages.SignInPage;
+import tests.BaseTest;
+
+public class CreateOrderTest extends BaseTest {
+    private final String restaurantName = "Gray Group";
+    private final String clientEmail = "nathansmith@test.com";
+    private final String clientPassword = "1111";
+    private final String menuItem1 = "Mustard-stuffed chicken";
+    private SignInPage signInPage;
+    private RestaurantsPage restaurantsPage;
+    private MenuPage menuPage;
+
+
+    @BeforeClass
+    public void preconditions() throws InterruptedException {
+        HomePage homePage = new HomePage(driver);
+        signInPage = homePage
+                .getHeaderGeneralPageComponent()
+                .signInAccess();
+        Thread.sleep(1000);
+        signInPage.setUserEmailInputField(clientEmail);
+        signInPage.setUserPasswordInputField(clientPassword);
+        signInPage.clickSignInBtn();
+        Thread.sleep(1000);
+        restaurantsPage = new RestaurantsPage(driver);
+    }
```

Comment:
Maybe we should use waits instead of sleeps

Reply:
You are right, I will fix it in the next sprint, if you don`t mind.

Reply:
Done